lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1753,6 +1759,12 @@ namespace Js
                 grfscr |= (fscrNoAsmJs | fscrNoPreJit);
             }
 
+            if (isLibraryCode)
+            {
+                grfscr |= fscrIsLibraryCode;
+                (*ppSourceInfo)->SetIsLibraryCode();
```

Comment:
we are not really using utf8 code path now though I'm adding usage for it later. here *ppSourceInfo is not created yet.